pocs/utils/images/__init__.py

```diff
@@ -24,6 +25,11 @@
 palette.set_under('k', 1.0)
 palette.set_bad('g', 1.0)
 
+palette = copy(plt.cm.inferno)
+palette.set_over('w', 1.0)
+palette.set_under('k', 1.0)
+palette.set_bad('g', 1.0)
```

Comment:
You've got two copies of this block of colourmap code.

Reply:
Ahh, will fix now.